MyCryptoHQ/MyCrypto
# Bug/Don't attempt to modify state with timeout when <WalletDecrypt /> is unmounted

Closes Core-43

### Description

Stops a timeout in `<WalletDecrypt />` from attempting to `setState` after the component has been unmounted

### Changes

* Added a type to the state interface to store the timeout ID
* Added logic to clear the timeout attempting to modify state when `<WalletDecrypt />` is unmounted using the stored timeout ID

### Steps to Test

1. Add debug code to display when a timeout is running and cleared
2. Login to a wallet 
3. Log out of the wallet to see if the timeout was cleared when the `<WalletDecrypt />` component was unmounted
